src/main/charts/agent/values.yaml

```diff
@@ -0,0 +1,233 @@
+## Atlassian Bamboo Data Center remote agent Helm values
+# 
+# HEADS UP! 
+#
+# A running and configured Bamboo server instance is a prerequisite for agent deployment!
+# Ensure that a valid security token and Bamboo server base URL have been supplied for the 
+# relevant properties below i.e:
+# - bamboo.securityToken
+# - bamboo.server
+#
+# Additionally, the Bamboo server must have; "Remote agent authentication" *disabled* and  
+# "Security token verification" *enabled*. Details on these features and how to enable
+# and disable them here:
+# https://confluence.atlassian.com/bamboo/agent-authentication-289277196.html
+##
+
+# -- The initial number of Bamboo agent pods that should be started at deployment time. 
+#
+replicaCount: 1
+
+# Image configuration 
+#
+image:
+
+  # -- The Bamboo agent Docker image to use
+  # https://hub.docker.com/r/atlassian/bamboo-agent-base
+  #
+  # TODO: This needs to be updated when docker porting work is complete
+  repository: atlssmith/bamboo-agent-base 
+  
+  # -- Image pull policy
+  #
+  pullPolicy: IfNotPresent
+  
+  # -- The docker image tag to be used - defaults to the Chart appVersion
+  #
+  tag: ""
+  
+# K8s ServiceAccount configuration. Give fine-grained identity and authorization 
+# to Pods
+#
+serviceAccount:
+
+  # -- Set to 'true' if a ServiceAccount should be created, or 'false' if it 
+  # already exists.
+  #
+  create: true
+  
+  # -- The name of the ServiceAccount to be used by the pods. If not specified, but 
+  # the "serviceAccount.create" flag is set to 'true', then the ServiceAccount name 
+  # will be auto-generated, otherwise the 'default' ServiceAccount will be used.
+  # https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server
+  #
+  name:
+  
+  # -- For Docker images hosted in private registries, define the list of image pull 
+  # secrets that should be utilized by the created ServiceAccount
+  # https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  #
+  imagePullSecrets: []
+  # - name: secretName
+
+# Bamboo agent configuration 
+#
+agent:
```

Comment:
I think we can pull this stanza to the top after `replicaCount` it will be the one part that needs to be changed so it is better when we lead with it.

Reply:
To be honest I think its better left where it is:

- Consistent positioning with the other charts
- Arguably the `image` stanza which comes after `replicaCount` takes priority
- and following on from the previous point so too does the `serviceAccount` stanza